### ISSUE TITLE
Format all <form> with <fieldset> instead of <table>

### ISSUE THREAD

**Issue by briri** [DMPRoadmap/roadmap#125]
Replace all of the existing <table> layouts on the forms with <fieldset>. Try to use the new dmproadmap.scss stylesheet.

**Comment by briri** [comment]
related to #230

**Comment by briri** [comment]
(copying over some conversation that happened in Slack)
@briri "I'm working on some of the UI issues that came out of our accessibility test last summer (specifically replacing the html tables that were causing problems).

I decided to try and follow the look of the new create plan page (#150). Let me know if this is a good approach for now.
![screen shot 2017-05-08 at 9 02 32 am](https://cloud.githubusercontent.com/assets/1204467/25860614/e4fc813e-3496-11e7-867e-62841718e5b2.png)

and here's the old look:
![screen shot 2017-05-08 at 9 06 33 am](https://cloud.githubusercontent.com/assets/1204467/25860631/faca3344-3496-11e7-92b0-1b37c89c1a7a.png)

I dropped the cancel button since it wasn't really doing anything besides bringing the user back to the view plans page. We can add it back if necessary. I also added some text below the email asking the user to also enter their password when changing the email address."

Some conversation then began about the workflow for changing one's email address and adding in the account recovery email address field. This particular task though is just about cleaning up the accessibility issues in the html form. We'll create a different ticket to record the desired changes to the edit profile page.

**Comment by briri** [comment]
(copying over some conversation that happened in Slack)
@briri: "and here's an example of one of the updated forms that are part of a larger page (like template/plan editing).
![screen shot 2017-05-08 at 11 02 09 am](https://cloud.githubusercontent.com/assets/1204467/25860828/a43a590e-3497-11e7-9d53-6453090c8fef.png)

@JEK-III: "This looks great. I do recommend moving the field labels to above the form elements (as you did on the profile page) instead of to the left. Although there are plenty of exceptions, top labels are generally considered best practice. In this case, because the labels are of such variable length, “ID,” for example is very far from the associated input field.
If that’s problematic in the short-term, I would right-justify the labels– which would make them harder for a user to scan, but give us a tighter association between the label and the field.

(None of which impacts the original underlying accessibility issue, which should be taken care of by what you’ve done already)"

@briri: "I'll right justify them then. I tried letting them fall above their input boxes originally but the visual transition from the read-only view of this info to editable form was a bit disorienting"

**Comment by briri** [comment]
(copying over some conversation that happened in Slack)
@briri: "the forms used to format the question-answer portion of the plan page doesn't use html tables but it does use ordered lists. Was that also an issue that was called  out on the accessibility report? If not I'll leave them as-is for now"

@rhu001: "I don't recall that being called as a major issue but in general tables and lists should be used for tabular data and lists of items respectively and not for form fields.

If there is time in this sprint to re-do them as normal form fields and not as an ordered list that would be preferable.

Unless there were numbers attached to each form field."

**Comment by briri** [comment]
Finished up Contact Us, Edit Profile, Create plan, edit plan details, answer questions, add/edit notes to plans, share plan forms.

Did not yet address the plan export page (this is being re-imagined by UX) and did not tackle any of the org admin pages (templates, guidance groups, org details, etc.)
